### DISCUSSION TITLE
Android location permission update

### DISCUSSION THREAD

**Discussion by Thangaraj-M** [fastlane/fastlane#17709] [Q&A]
In Fastlane, is there any option available to declare location permission for Android?
https://support.google.com/googleplay/android-developer/answer/9214102

**Comment by Thangaraj-M** [comment]
It is required to update the location permission declaration for the new apps in google from Jan 18, 2021. Can we update this using Fastlane?

**Reply by Thangaraj-M** [reply]
Could you please someone help for this?

**Comment by Pouros87** [comment]
Your app's manifest.xml having these permission to access information from your's device but you don't have privacy policy link while submitting on the play store. so you getting this warning [myprepaidcenter.com](https://myprepaidcenter.net/)

**Reply by Thangaraj-M** [reply]
> Your app's manifest.xml having these permission to access information from your's device but you don't have privacy policy link while submitting on the play store. so you getting this warning [myprepaidcenter.com](https://myprepaidcenter.net/)

No. I am not asking this. I am asking whether any option present in Fastlane to update the location permission declaration in play console

**Comment by Thangaraj-M** [comment]
Getting the below  error when running the supply command. It is because of the location permission declaration is not given. 
**Google Api Error: forbidden: Release artifacts require permissions that are missing from permission declaration. - Release artifacts require permissions that are missing from permission declaration.**

**Comment by josehernandezv** [comment]
Any updates on this?
